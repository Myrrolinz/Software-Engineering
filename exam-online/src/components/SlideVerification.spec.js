import { shallowMount } from '@vue/test-utils';
import SlideVerification from './SlideVerification';





describe('<SlideVerification/>', () => {
	const wrapper = shallowMount(SlideVerification);

	// 快照测试
	it('snapshot测试', () => {
		const wrapper2 = shallowMount(SlideVerification);
		expect(wrapper2.html()).toMatchSnapshot()
		wrapper2.destroy()
    })

	
  });

  /*
  it("slide correctly", () => {
		// 滑动 slide
		wrapper.find('slide-verification').trigger('check-result');
	  });
  		// 可以立即获取 msg 最新的值，不再需要 wrapper.vm.$nextTick();
		expect(wrapper.find('h1').text())
		  .toEqual('new message');
		  */
