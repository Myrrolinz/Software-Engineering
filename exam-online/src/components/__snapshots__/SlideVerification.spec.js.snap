// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SlideVerification/> snapshot测试 1`] = `
"<div class=\\"drag\\">
  <div class=\\"drag_bg\\"></div>
  <div class=\\"drag_text\\">拖动滑块验证</div>
  <div class=\\"handler handler_bg\\" style=\\"position: absolute; top: 0px; left: 0px;\\"></div>
</div>"
`;
